src/components/Aside/Sidebar.tsx

```diff
@@ -0,0 +1,148 @@
+import React, { useState, useEffect } from 'react';
+import styled from 'styled-components';
+import { AiOutlineDown, AiOutlineUp } from 'react-icons/ai';
+
+import { firestore } from '../../shared/api/firebase';
+import { Link, useParams } from 'react-router-dom';
+import { theme } from '../../styles/Theme';
+
+interface MenuSectionProps {
+  id: number;
+  title: string;
+  items: MenuItem[];
+  url?: string;
+}
+
+interface MenuItem {
+  text: string;
+  timeStamp: string;
+  url: string;
+}
+
+interface ItemContainerProps {
+  $isUrl: boolean;
+}
+
+const SidebarContainer = styled.div`
+  min-width: 256px;
+  height: 100vh;
+  background-color: ${theme.blueBg3};
+  color: ${theme.gray300};
+  padding: 24px 20px 0px;
+  position: fixed;
+`;
+
+const MenuContainer = styled.div`
+  margin-bottom: 32px;
+`;
+
+const TitleContainer = styled.div`
+  height: 28px;
+  padding: 5px 8px;
+  font-size: ${theme.textStyles.caption.fontSize};
+  line-height: ${theme.textStyles.caption.lineHeight};
+  border-bottom: 1px solid #e5e9ed;
+  display: flex;
+  justify-content: space-between;
+  cursor: pointer;
+`;
+
+const ItemContainer = styled.div<ItemContainerProps>`
+  background-color: ${(props) => (props.$isUrl ? theme.blueBg1 : '')};
```

Comment:
```suggestion
  ${({$isUrl, theme}) => $isUrl && `background-color: ${theme.blueBg1}` }
```

이렇게 해야, 불필요한 css가 적용이 되지 않겠죠?!
지금은 
background-color: ''

이렇게 되어서 적용이 안 되는 css가 불필요하게 CSS Tree에 읽히게 됩니다.